.github/workflows/build.yml

```diff
@@ -35,10 +35,20 @@ jobs:
         env:
           TARGET: ${{ matrix.target }}
           DOCKER_TAG: ${{ matrix.container }}
+          SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+
 
       - name: Rename Binary
         run: mv target/*/release/sentry-cli sentry-cli-Linux-${{ matrix.arch }}
 
+      - name: Upload Symbols
+        env:
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+        run: sentry-cli-Linux-${{ matrix.arch }} upload-dif sentry-cli-Linux-${{ matrix.arch }} --org <FILL IN ORG> --project <FILL IN PROJECT>
```

Comment:
NOTE: Either `<FILL IN ORG>` and `<FILL IN PROJECT>` or use same as on line 105

Reply:
ops! thanks for noticing!